com.unity.multiplayer.mlapi/Runtime/Core/NetworkManager.cs

```diff
@@ -35,6 +35,9 @@ public class NetworkManager : MonoBehaviour, INetworkUpdateSystem, IProfilableTr
 #if UNITY_2020_2_OR_NEWER
         // RuntimeAccessModifiersILPP will make this `public`
         internal static readonly Dictionary<uint, Action<NetworkBehaviour, NetworkSerializer, __RpcParams>> __ntable = new Dictionary<uint, Action<NetworkBehaviour, NetworkSerializer, __RpcParams>>();
+        // IF TOOLS
+        // For debug and profiling purpose only
+        internal static readonly Dictionary<uint, string> __rpc_name_table = new Dictionary<uint, string>();
```

Comment:
Is there a way to figure out the name of the RPC as it happens, rather than keeping a map around?

Asking mostly out of curiosity, because I figure if this the solution we ended up with, it's probably the only way.

Reply:
There's no other way really. We will use this to retrieve the name when invoking RPC.

Reply:
I just realized it'd be nicer to call `__ntable` as `__rpc_table` too :)

Reply:
nah, but I will do that soon.